community/prioritization_response.md

```diff
@@ -0,0 +1,58 @@
+## Overview
+
+Given that we are a small functional team that is meant to maintain :wrench:, support ::phone::, care for :heart:, and cultivate :sunflower: our amazing Octokit :octocat: and Terraform community we often have to make decisions that are best for the entire community.
+We don't take this act of prioritization lightly nor are we haphazard about it. We want to honor the time and thought that everyone has put into all of the SDKs that we have come to know and :heart:.
+
+Add that to the number of repos that we need to maintain (70+) and weekly interactions (500-1000 +/-) that we end up having with everyone I am sure it's clear why we have to define how we **prioritize** and our commitment to respond (think of an **SLA** of sorts) to those priorities.
+
+### What are the Priorities?
+
+In each of the repositories we own you'll see the same [labels](https://github.com/octokit/octokit.net/labels) for priority.  
+
+**They are:**  
+`Priority: Critical`, `Priority: High`, `Priority: Normal`, `Priority: Low`
+
+Incidentally, these map directly to the priorities set in our [Octokit Active community board](https://github.com/orgs/octokit/projects/10/views/4).
+
+| [Repo label](https://github.com/octokit/octokit.net/labels)   |      [Community board](https://github.com/orgs/octokit/projects/10/views/4) | Example |
+|----------|:-------------:|:-----------|
+| Priority: Critical | Urgent | Any security issues - Data leaks - Broken or non functioning SDK that has been published - Any stop work scenarios for users (where rollback is not possible)|
+| Priority: High | High | Any broken SDK methods - Deprecated or broken features - Critical dependency updates - Pull Requests that have been completed, are active, and are passing CI - Any issues that have been reported multiple times / impacting multiple users |
```

Comment:
Are these dashes supposed to be a bulleted list or not? They're not [rendering](https://github.com/octokit/.github/blob/f68e4bf2f6431a45099a110cf289b40a270a84b9/community/prioritization_response.md) as one right now.

Reply:
They were just meant to be separators.  If you come up with a better layout let me know!

Reply:
That's a bit of a tough one. I'm :+1: to leave them as they are right now; I don't have any better ideas.